.vscode/launch.json

```diff
@@ -0,0 +1,240 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "sentiment-wsl",
```

Comment:
Why do we need wsl-specific tasks here? People should just enter the directory from within WSL if that's what they want.

Reply:
It's makes it easier to debug from the root. You need a separate config for WSL if you are debugging in that environment using VSCODE.

Reply:
Yes

Reply:
So I tried this out locally - my experience is that `externalTerminal` didn't seem to work at all, regardless of whether I was running in WSL or not. Instead, only `integratedTerminal` worked across both.

So I think that matches what I recommended elsewhere - that all the WSL-oriented tasks should be the defaults, and there shouldn't be any sort of variant. If that's not matching your experience, we can jump on a call and run through it.

Reply:
Yes, let’s sync on Monday. I want to check the configuration on windows.